Crash report 2.7.2 - NPE in ImageHelper

``````
java.lang.RuntimeException: An error occured while executing doInBackground()
at android.os.AsyncTask$3.done(AsyncTask.java:278)
at java.util.concurrent.FutureTask$Sync.innerSetException(FutureTask.java:273)
at java.util.concurrent.FutureTask.setException(FutureTask.java:124)
at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:307)
at java.util.concurrent.FutureTask.run(FutureTask.java:137)
at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:208)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1076)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:569)
at java.lang.Thread.run(Thread.java:856)
Caused by: java.lang.NullPointerException
at org.wordpress.android.util.ImageHelper.getImageSize(ImageHelper.java:41)
at org.wordpress.android.util.ImageHelper.getThumbnailForWPImageSpan(ImageHelper.java:354)
at org.wordpress.android.ui.posts.EditPostContentFragment.addMedia(EditPostContentFragment.java:1027)
at org.wordpress.android.ui.posts.EditPostContentFragment.access$600(EditPostContentFragment.java:93)
at org.wordpress.android.ui.posts.EditPostContentFragment$processAttachmentsTask.doInBackground(EditPostContentFragment.java:652)
at org.wordpress.android.ui.posts.EditPostContentFragment$processAttachmentsTask.doInBackground(EditPostContentFragment.java:637)
at android.os.AsyncTask$2.call(AsyncTask.java:264)
at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:305)
... 5 more```
``````

--- a/src/org/wordpress/android/util/MediaUtils.java
+++ b/src/org/wordpress/android/util/MediaUtils.java
@@ -413,7 +413,12 @@ public class MediaUtils {
 
         if (TextUtils.isEmpty(mimeType)) {
             try {
-                String filePathForGuessingMime = mediaFile.getPath().contains("://") ? mediaFile.getPath() : "file://"+mediaFile.getPath();
+                String filePathForGuessingMime;
+                if (mediaFile.getPath().contains("://")) {
+                    filePathForGuessingMime = Uri.encode(mediaFile.getPath(), ":/");
+                } else {
+                    filePathForGuessingMime = "file://"+ Uri.encode(mediaFile.getPath(), "/");
+                }
                 URL urlForGuessingMime = new URL(filePathForGuessingMime);
                 URLConnection uc = urlForGuessingMime.openConnection();
                 String guessedContentType = uc.getContentType(); //internally calls guessContentTypeFromName(url.getFile()); and guessContentTypeFromStream(is);

